lib/bindings/src/breez_sdk_liquid.udl

```diff
@@ -614,6 +631,7 @@ interface SdkEvent {
     PaymentRefundPending(Payment details);
     PaymentSucceeded(Payment details);
     PaymentWaitingConfirmation(Payment details);
+    PaymentWaitingUserAction(Payment details);
```

Comment:
Since we have fetch fees and accept fees (explicit relates to fee) perhaps here we should also be more specific?
`PaymentWaitingFeeAcceptance`? 
Then the following `fetch_swap_updated_fees` is more intuitive. What do you think?

Reply:
Let's go with that. I think it's plausible we get other types of required user actions in the future, but I cannot think of any right now. If it happens, shouldn't be too much overhead to deprecate the specific event in favor of a slightly more generic one.